tests/testthat/test-exporters.R

```diff
@@ -151,18 +151,6 @@ test_that("export_as_txt maintains horizontal separator from table", {
   expect_equal(out, paste0(rep("=", nchar(out)), collapse = ""))
 })
 
-test_that("export_as_txt works with prov footer and page numbers", {
-  dfmf <- basic_matrix_form(mtcars)
-  prov_footer(dfmf) <- c("my", "prov", "footer")
-
-  set_default_page_number("page {i} of {n}")
-  expect_silent(res <- export_as_txt(dfmf, paginate = TRUE, cpp = 60))
-  set_default_page_number(NULL)
-
-  expect_true(grepl("page 1 of 2", res))
-  expect_true(grepl("page 2 of 2", res))
-})
-
```

Comment:
This needs to be reinstated. We added it for the issue we described yesterday

Reply:
I didn't (intentionally) delete this, when was it added?

Reply:
I think yesterday! It was part of a small but very important hot-fix for page numbers. As we said yesterday, we will have to merge the two logics and move the page numbering at a later step.

Reply:
I have different code that passes all the checksincluding this reinstated test in the branch now